src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,55 @@
+package lotto.domain;
+
+import lotto.exception.InvalidInputException;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+public class Lotto {
+    private static final String INVALID_SIZE_OF_WINNING_NUMBERS = "번호의 갯수가 올바르지 않습니다.";
+    public static final int SIZE = 6;
+
+    private Set<LottoNumber> numbers;
+
+    private Lotto(Set<LottoNumber> numbers) {
+        validateLottoSize(numbers);
+        this.numbers = numbers;
+    }
```

Comment:
```suggestion
    private Lotto(Set<LottoNumber> numbers) {
        validateLottoSize(numbers);
        this.numbers = numbers;
    }
    
    public static Lotto of(Set<LottoNumber> numbers) {
        return new Lotto(numbers);
    }
```
기본 생성자를 위한 `of` 메소드도 필요하지 않을까요?